modular_skyrat/modules/customization/modules/clothing/under/security.dm

```diff
@@ -48,3 +48,94 @@
 	heat_protection = CHEST|ARMS
 
 // SEC GENERAL
+
+// PRISONER
+/obj/item/clothing/under/rank/protcustpskirt
+	name = "protective custody prisoner jumpsuit"
```

Comment:
You could just make all these a subtype of the first

Reply:
what would that do?

Reply:
Means you don't have to copy pasta code and it's easier to find and spawn, better code practice

Reply:
ahh, on it

Reply:
sorted